cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -387,5 +387,27 @@ describe('Moderation Facility Submission Form', () => {
             cy.get('[data-testid="submission-form-mapLongitude"]').find('input').clear().type('Not Number Longitude').realPress('Tab')
             cy.get('[data-testid="submission-form-mapLongitude"]').find('p', { timeout: 10000 }).should('exist').contains('Invalid Longitude')
         })
+
```

Comment:
Test to make sure the error message for retry is shown on server error sent back